backend/src/tasks/intrigue-ident.ts

```diff
@@ -0,0 +1,49 @@
+import { Domain } from '../models';
+import { CommandOptions } from './ecs-client';
+import getLiveWebsites from './helpers/getLiveWebsites';
+import PQueue from 'p-queue';
+import * as buffer from 'buffer';
+import { spawnSync } from 'child_process';
+
+const intrigueIdent = async (domain: Domain): Promise<void> => {
+  console.log('Domain', domain.name);
+  for (const service of domain.services) {
+    if (service.port !== 80 && service.port !== 443) {
+      continue;
+    }
+    const url =
+      service.port === 443 ? `https://${domain.name}` : `http://${domain.name}`;
+    const { stdout, stderr, status } = spawnSync('intrigue-ident', [
```

Comment:
Let's add the `-c` flag, which returns some other fun useful info:

![40c84f3f-eff1-4ad6-a2cb-09d5ba6082b7](https://user-images.githubusercontent.com/7061968/92004716-2dac2880-ed08-11ea-90ba-b94f3401b3fb.jpg)

Reply:
We actually already receive this in the JSON output. We store it in `intrigueIdentResults.content`.